src/components/NcAvatar/NcAvatar.vue

```diff
@@ -384,11 +381,8 @@ export default {
 	},
 	computed: {
 		avatarAriaLabel() {
-			if (!this.hasMenu) {
-				return
-			}
```

Comment:
I believe this should be kept as `aria-label` should only be present on interactive elements as documented in https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label